src/treq/testing.py

```diff
@@ -402,7 +402,7 @@ class RequestSequence(object):
 
     - ``code`` is an integer representing the HTTP status code to return.
     - ``headers`` is a dictionary mapping :class:`bytes` to :class:`bytes` or
-      :class:`list` of :class:`bytes`.
+      :class:`str`. Note that the value is *not* a list.
```

Comment:
What *is* the format for header items with multiple values?

Reply:
They cannot be represented in this API. :-/ You could send the semantic equivalent with a comma... _\*mumble mumble cookies\*_...

Reply:
Ouch.  Okay.